src/control/conf.c

```diff
@@ -556,13 +589,37 @@ int dt_confgen_get_int(const char *name, dt_confgen_value_kind_t kind)
 {
   const char *str = dt_confgen_get(name, kind);
   const float value = dt_calculator_solve(1, str);
```

Comment:
do you confirm that this will return isnan() when str is NULL?

Reply:
Maybe a better option is to explicitly check that the value exists, at least the intention will be clear when reading code.

So I would suggest calling dt_confgen_value_exists() and take proper action.

Reply:
`dt_calculator_solve` first lines is:

```c
if(formula == NULL || *formula == '\0') return NAN;
```

so I think It's fine to feed null into it. Do you still recomend to check with `dt_confgen_value_exists`?

Reply:
Yes otherwise one will have to look at the implementation of dt_calculator_solve() to be sure code is ok, just reading this on this PR doesn't make this clear. So to me a clear and explicit check will ease maintenance.

Reply:
Done!